bigtable/docs/snippets.py

```diff
@@ -160,6 +174,205 @@ def test_bigtable_create_app_profile():
     app_profile.delete(ignore_warnings=True)
 
 
+def test_bigtable_create_table():
+    # [START bigtable_create_table]
+    from google.cloud.bigtable import Client
+    from google.cloud.bigtable import column_family
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+
+    # Create table with Column families.
+    table1 = instance.table("table_id1_create")
+    # Define the GC policy to retain only the most recent 2 versions.
+    max_versions_rule = column_family.MaxVersionsGCRule(2)
+    table1.create(column_families={'cf1': max_versions_rule})
+
+    # Create table without Column families.
+    table2 = instance.table("table_id2_create")
+    table2.create()
+    # [END bigtable_create_table]
+    assert table1.exists()
+    assert table2.exists()
+    table1.delete()
+    table2.delete()
+
+
+def test_bigtable_sample_row_keys():
+    # [START bigtable_sample_row_keys]
+    from google.cloud.bigtable import Client
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+
+    table = instance.table("table_id1_samplerow")
+    table.create()
+
+    data = table.sample_row_keys()
+    for element in data:
+        offset_bytes = str(element).split(":")[1].strip()
+    # [END bigtable_sample_row_keys]
+    assert offset_bytes == '805306368'
+    table.delete()
+
+
+def test_bigtable_create_column_family():
+    # [START bigtable_create_column_family]
+    from google.cloud.bigtable import Client
+    from google.cloud.bigtable import column_family
+
+    client = Client(admin=True)
+    instance = client.instance(INSTANCE_ID)
+
+    table = instance.table(TABLE_ID)
+    column_family_id = 'column-family-id1'
+
+    gc_rule = column_family.MaxVersionsGCRule(1)
+    column_family = table.column_family(column_family_id, gc_rule=gc_rule)
+    column_family.create()
+    # [END bigtable_create_column_family]
+
+    col_fams = table.list_column_families()
+    assert column_family_id in col_fams
+
+
+def test_bigtable_mutations_batcher_read_rows():
```

Comment:
The name is a bit confusing IMHO, perhaps you can do `mutate_rows` -> 'read_row` -> `read_rows` -> 'drop_by_prefix` -> 'truncate` in one example ([link](https://gist.github.com/AVaksman/ef78ccff8c9b98461c674dcbf9320e53)) and batcher in a separate example.

Reply:
Done